### PR TITLE
119-Add-the-FAQ

### PR DESCRIPTION
**Pull request by davidstanke**
This PR adds a FAQ, with content from two sources:
1. migration of content from devops-reseach.com/faq.html
2. new content about components of dora.dev, in particular, the Quick Check.

### CODE REVIEW

**Comment by davidstanke** [comment]
Preview is available at https://draft-dora-dev--pr132-cf2ozhtl.web.app/

File: hugo/assets/scss/faq.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+
+styles for UI widgets such as popovers, modals, tabs
+
```
Comment (davidstanke) [inline hugo/assets/scss/faq.scss:4]: Update this comment… it’s copied over from widgets.scss

**Comment by nathenharvey** [comment]
LGTM!

**Pull request merged**